updateStableIds/src/main/java/org/reactome/release/updateStableIds/UpdateStableIds.java

```diff
@@ -94,9 +95,11 @@ public static void stableIdUpdater(MySQLAdaptor dbaSlice, MySQLAdaptor dbaPrevSl
 							logger.info("StableIdentifer has already been updated during this release");
 						}
 					}
-				} catch (Exception e) {
+				} catch (StackOverflowError e) {
```

Comment:
StackOverflowError seems oddly specific. I'm just curious: what is triggering it?

Reply:
It shouldn't happen in reality, but it came up in testing due to a recursive call